server/shared/src/main/scala/org/http4s/server/websocket/WebSocketBuilder2.scala

```diff
@@ -177,7 +180,7 @@ sealed abstract class WebSocketBuilder2[F[_]: Applicative] private (
 }
 
 object WebSocketBuilder2 {
-  private[http4s] def apply[F[_]: Applicative](
+  def apply[F[_]: Applicative](
```

Comment:
It almost seems like we should deprecate this constructor in favor of the other.

Reply:
I think we could deprecate it, didn't remove it because the old `WebSocketBuilder` still uses it, but that's deprecated so... What should I set as the version on the deprecated message? 0.23.6?

Reply:
I think we're onto 0.23.15?